web_widget_color/static/src/js/widget.js

```diff
@@ -16,8 +16,20 @@ odoo.define('web.web_widget_color', function(require) {
 
         _renderEdit: function() {
             this.$input = this.$el.find('input');
-            this.jscolor = new jscolor(this.$input[0], {hash:true, zIndex: 2000});
+            this.jscolor = new jscolor(this.$input[0], {
```

Comment:
I believe we shouldn't create unnecessary difs just for formatting.

Reply:
It fixes lint errors.